library/ssl_tls12_client.c

```diff
@@ -39,7 +39,10 @@
 #if defined(MBEDTLS_USE_PSA_CRYPTO)
 #include "mbedtls/psa_util.h"
 #include "psa/crypto.h"
-#endif /* MBEDTLS_USE_PSA_CRYPTO */
+#define HASH_MAX_SIZE PSA_HASH_MAX_SIZE
+#else /* !defined(MBEDTLS_USE_PSA_CRYPTO) */
+#define HASH_MAX_SIZE MBEDTLS_MD_MAX_SIZE
+#endif /* !defined(MBEDTLS_USE_PSA_CRYPTO) */
```

Comment:
The comments on `#else`/`#endif` are unnecessary as the `#define` is so close, and they severely impact readability (yes, I know the one on `#endif` was already there)

```suggestion
#define HASH_MAX_SIZE PSA_HASH_MAX_SIZE
#else
#define HASH_MAX_SIZE MBEDTLS_MD_MAX_SIZE
#endif
```

Reply:
Thank you. I will clean up the extra comment.